SubmittedAssignments/Sanjeevanee_Balure_38/preview.html

```
@@ -0,0 +1,188 @@
+			</div>
+			<div>
+				<img src="https://th.bing.com/th/id/OIP.mUyAb4urCggDzQ6a6aBA6wHaGL?rs=1&pid=ImgDetMain" class="side-image lettuce-1">
+			</div>
+			<div>
+				<img src="https://st2.depositphotos.com/3210553/6949/i/950/depositphotos_69499281-stock-photo-tomato-slice-isolated-on-white.jpg" class="side-image tomato-slice-1">
+			</div>
+			
+		</div>
+
+		<!-- Menu Card -->
+		<menu-card data='{
```

HTMLHint (1.5.0)

[error] The value of attribute [ data ] must be in double quotes.

(attr-value-double-quotes)